packages/react-relay/relay-hooks/QueryResource.js

```diff
@@ -466,16 +466,10 @@ class QueryResourceImpl {
           }
           const observerStart = observer?.start;
           if (observerStart) {
-            const subscriptionWithConditionalCancelation = {
+            observerStart({
               ...subscription,
-              unsubscribe: () => {
-                // Only live queries should have their network requests canceled.
-                if (operationIsLiveQuery(operation)) {
-                  subscription.unsubscribe();
-                }
-              },
-            };
-            observerStart(subscriptionWithConditionalCancelation);
+              unsubscribe: () => {},
```

Comment:
Do live queries still get unsubscribed through some other means?

Reply:
Yeah it's managed by the `.retain()` stuff and is also ensured with the test added (the `.toThrow()` part)